src/librustc_driver/lib.rs

```diff
@@ -1169,18 +1183,24 @@ pub fn init_rustc_env_logger() {
 }
 
 pub fn main() {
+    let start = Instant::now();
     init_rustc_env_logger();
+    let mut callbacks = TimePassesCallbacks::default();
     let result = report_ices_to_stderr_if_any(|| {
         let args = env::args_os().enumerate()
             .map(|(i, arg)| arg.into_string().unwrap_or_else(|arg| {
                 early_error(ErrorOutputType::default(),
                             &format!("Argument {} is not valid Unicode: {:?}", i, arg))
             }))
             .collect::<Vec<_>>();
-        run_compiler(&args, &mut DefaultCallbacks, None, None)
+        run_compiler(&args, &mut callbacks, None, None)
     }).and_then(|result| result);
-    process::exit(match result {
+    let exit_code = match result {
         Ok(_) => EXIT_SUCCESS,
         Err(_) => EXIT_FAILURE,
-    });
+    };
+    // The extra `\t` is necessary to align this label with the others.
+    set_time_depth(0);
```

Comment:
Why is this `\t` necessary?

Reply:
The comment immediately above explains why. If the comment is not clear, please suggest an alternative wording.

Reply:
I don't understand the reason. The other calls have no extra `\t`, so why is it needed here?

Reply:
Sample output with the extra `\t` (with 8-space tabs):
```
  time: 0.000; rss: 101MB       codegen
  time: 0.000; rss: 101MB       serialize work products
  time: 0.000; rss: 101MB       linking
time: 0.049; rss: 77MB          total
```
Without the extra `\t`:
```
  time: 0.000; rss: 101MB       codegen
  time: 0.000; rss: 101MB       serialize work products
  time: 0.000; rss: 101MB       linking
time: 0.049; rss: 77MB  total
```
(The tabs were converted to spaces when I pasted them into this comment, but the layout is indicative.)

Reply:
That is kind of supposed to happen? =P